akka-persistence/src/main/scala/akka/persistence/SnapshotProtocol.scala

```diff
@@ -123,6 +135,13 @@ object SnapshotSelectionCriteria {
   /**
    * Java API.
    */
+  def create(maxSequenceNr: Long, maxTimestamp: Long,
+             minSequenceNr: Long, minTimestamp: Long) =
+    SnapshotSelectionCriteria(maxSequenceNr, maxTimestamp, minSequenceNr, minTimestamp)
```

Comment:
there was already a `def create(maxSequenceNr: Long, maxTimestamp: Long)`
we can't provide all permutations (default values)